### ISSUE TITLE
BUG: Parameter is not being recognized in WITH statement

### ISSUE THREAD

**Issue by aarbmx6s** [beekeeper-studio/beekeeper-studio#2052]
**Describe the bug**

The $1 parameter is not being recognized within the WITH clause:
```
WITH test AS (SELECT $1)
SELECT * FROM test;
```

<img width="589" alt="image" src="https://github.com/beekeeper-studio/beekeeper-studio/assets/22832351/bbafbbda-817c-44dd-bbcf-fb7457b00224">

**To Reproduce**
1. Connect to postgesql 
2. Run the query from above
5. See error

**Expected behavior**
The manager recognizes such parameters

**Version Information (please complete the following information):**
 - OS: Mac
 - App Version: 4.1.13 (4.1.13)
 - Database type and version: Posgresql 13

**Additional context**
Parameters are recognized properly for queries like:
```
SELECT $1;
```

**Comment by not-night-but** [comment]
Yes there was an issue in `sql-query-identifier` where CTE params were not identified, this is fixed now.
